queries.yaml

```diff
@@ -122,3 +122,67 @@ pg_database:
     - size:
         usage: "GAUGE"
         description: "Disk space used by the database"
+
+pg_stat_statements:
+  query: "SELECT query, calls, total_time, min_time, max_time, mean_time, stddev_time, rows, shared_blks_hit, shared_blks_read, shared_blks_dirtied, shared_blks_written, local_blks_hit, local_blks_read, local_blks_dirtied, local_blks_written, temp_blks_read, temp_blks_written, blk_read_time, blk_write_time FROM pg_stat_statements"
+  metrics:
+    - query:
+        usage: "LABEL"
+        description: "Query class"
+    - calls:
+        usage: "COUNTER"
+        description: "Number of times executed"
+    - total_time:
```

Comment:
This, and other `..._time` stuff. Shouldn't they all have units in the suffix of the metric? I.e. `pg_stat_statements_total_time_seconds` , so `SELECT ... , total_time / 1000 as total_time_seconds, ...`

See https://prometheus.io/docs/practices/naming/#metric-names and https://github.com/wrouesnel/postgres_exporter/issues/261